backend/src/test/java/gov/cdc/usds/simplereport/api/converter/FhirConverterTest.java

```diff
@@ -304,4 +313,165 @@ void null_convertToTribalAffiliation() {
     assertThat(convertToTribalAffiliationExtension((String) null)).isNull();
     assertThat(convertToTribalAffiliationExtension((List<String>) null)).isNull();
   }
+
+  @Test
+  void testEvent_convertToDiagnosticReport() {
+    var testEvent =
```

Comment:
Same feedback about these duplicate TestEvents as on the other PR, but I almost wonder if it's worth creating a `TestFactoryBuilder` class that would have methods to create null Person, Facility, etc without actually saving them to the database. (Normally I'd suggest using `TestDataFactory`, but that requires a backing database)